commands/local_proxy_domain_attach.go

```diff
@@ -48,7 +51,16 @@ var localProxyAttachDomainCmd = &console.Command{
 		if err != nil {
 			return err
 		}
-		if err := config.AddDirDomains(projectDir, c.Args().Tail()); err != nil {
+
+		var domains []string
+		if strings.HasPrefix(c.Args().Slice()[0], ".") {
+			config.TLD = strings.TrimPrefix(c.Args().Slice()[0], ".")
+			domains = c.Args().Slice()[1:]
+		} else {
+			domains = c.Args().Slice()
+		}
+
+		if err := config.AddDirDomains(projectDir, domains); err != nil {
```

Comment:
I don't think this makes sense to allow attaching and changing the TLD at the same time: this cover a quite small use case and it could be confusing for the end-user letting them think this TLD is only for the specific project (which is not the case).

Reply:
You are right. I reverted the file back to it's previous state.